CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+### v1.0...v1.1
```

Comment:
I think it needs a title with the new version and the release date somewhere around here?

```suggestion
## v1.1 - 2023-07-18

### v1.0...v1.1
```